en/docs/adv_examples/bert.md

```diff
@@ -1,48 +1,48 @@
 
-## 模型概述
-BERT(Bidirectional Encoder Representations from Transformers)是NLP领域的一种新型预训练模型。本案例中，基于论文[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805)实现了BERT模型的OneFlow版本。本案例中，基于论文[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805)实现了BERT模型的OneFlow版本。
+## Summary
+BERT(Bidirectional Encoder Representations from Transformers) is a technique for NLP. In our case, we implements BERT based on the paper [BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805) using OneFlow.
 
-### 模型架构
+### Model
 | **Model** | **Hidden layers** | **Hidden unit size** | **Attention heads** | **Feedforward filter size** | **Max sequence length** | **Parameters** |
 |:---------:|:-----------------:|:--------------------:|:-------------------:|:---------------------------:|:-----------------------:|:--------------:|
 | BERTBASE  |    12 encoder     |         768          |         12          |          4 x  768           |           512           |      110M      |
 
-BERT在实际应用中往往分为两步：
+There are commonly two steps in BERT:
 
-* 首先，预训练得到BERT语言模型；
+* First, BERT pre-trained is obtained by pre-training;
 
-* 然后，为满足下游应用，在得到的BERT语言模型的基础上，多加一层网络，并进行微调，得到下游应用。
+* Then, on the basis of the obtained pre-trained model, an additional layer of network is added and finetuned to get the downstream application.
```

Comment:
Then, on the basis of the obtained pretrained model, an additional layer of network is added and finetuned to get the downstream application.